docs/src/tutorials/tutorial_morphing.md

```diff
@@ -0,0 +1,188 @@
+# **Tutorial 9:** Morphing `Javis` Objects   
+
+There are multiple ways to morph an object in Javis.
+
+- Using the older `morph_to(::Function)` method. Very nice at matching the shapes for morphing . Has limitations on what the morphing function should contain.
+- Using (New) `morph_to(::Object)` method. Any Object can be morphed to any other object using this method.
+- Using `morph_to_fn(::Function)` method. Similar to `morph_to(::Object)` but morphs to function instead. Can morph an object to a function that contains Luxor calls to draw what it should morphed into.
```

Comment:
I think it's quite confusing why we have `morph_to(::Function)` and `morph_to_fn`. Can't they be combined?

Reply:
was not sure how to write a method which doesn't overwrite the existing method for the old morph